python/lancedb/db.py

```diff
@@ -333,3 +341,11 @@ def drop_table(self, name: str, ignore_missing: bool = False):
         except FileNotFoundError:
             if not ignore_missing:
                 raise
+
+    def drop_database(self):
+        try:
+            filesystem, path = fs_from_uri(self.uri)
+            shutil.rmtree(path)
```

Comment:
We would like to use `filesystem` to run deletion.  pyarrow filesystem can support cloud storages, like, S3 or GCS.

Reply:
No, I wanted to check if there are no tables in the directory . i.e. if the user has deleted all the tables but  he called drop_database.